makerdao/governance-portal-v2
# Fix/older poll types missing

### What does this PR do?

Fixes an issue where older polls are missing a poll type by adding an unknown poll type

### Steps for testing:

Browse through vote histories for various voters.

Prior to June 2020, there is an unknown vote type. These should not show any plurality votes since we don't have that information, but they should still render the persons vote.

### Screenshots (if relevant):

<img width="755" alt="Screen Shot 2021-09-16 at 1 13 09 PM" src="https://user-images.githubusercontent.com/5225766/133602323-9087f4fd-ac02-41c0-b053-560b09406888.png">

### Any additional helpful information?:

Also adds unit tests for some components using vote type

### Add a GIF:
![](https://media.giphy.com/media/OHDGDRDfz2JD3EBVoK/giphy.gif)
